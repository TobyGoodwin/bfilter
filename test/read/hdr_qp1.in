Subject: yes =?iso-8859-1?q?Just_=A35!?= this is valid

this is the body
