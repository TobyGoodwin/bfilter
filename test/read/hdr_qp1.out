{ yes Just Â£5! this is valid}
{this is the body}
